quadcontrol/config.c

```diff
@@ -0,0 +1,266 @@
+/*
+ * Phoenix-Pilot
+ *
+ * Flight scenario parser
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-Pilot software
+ *
+ * %LICENSE%
+ */
+
+#include "config.h"
+#include <parser.h>
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+
+#define SCENARIO_TYPES_NB   7
+#define SCENARIO_MAX_FIELDS 3
+
+#define BASE 10
+
+
+enum { config_scenarioID = 0, config_pidID, config_end };
```

Comment:
To make a shorter references in the code:
```suggestion
enum { cfg_scenarioID = 0, cfg_pidID, cfg_end };
```